docs/architecture/blazor-for-web-forms-developers/pages-routing-layouts.md

```diff
@@ -212,17 +218,17 @@ In Blazor, you handle page layout using layout components. Layout components inh
 
 When the page with a layout is rendered, the page is rendered within the contents of the specified layout at the location where the layout renders its `Body` property.
 
-To apply a layout to a page, use the `@layout` directive.
+To apply a layout to a page, use the `@layout` directive:
 
 ```razor
```

Comment:
I haven't seen `razor` identifier before in docs. I doubt that such an identifier is supported.
/cc @mairaw

Reply:
When it's not supported by default, docs will just display that identifier with no coloring.

Reply:
It's not currently supported. I submitted a request to engineering a few months back. The closest dev lang to it is "CSHTML".